src/app/login/login.component.css

```diff
@@ -0,0 +1,229 @@
+body,
+html {
+    height: 100%;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    background-color: #161622;
+    color: #fff;
+    margin: 0;
+    padding: 0;
+}
+
+.button-center {
+    display: flex;
+    justify-content: center;
+}
+
+.border-login {
+    margin-top: 50px; 
+    border: 1px solid #fff;  
+}
+
+.page-container {
+    overflow-y: auto;
+    background-color: #003F48;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.button-container {
+    background-color: #003F48;
+    padding: 10px;
+}
+
+.button-container button {
+    background-color: #6c757d;
+    color: #fff;
```

Comment:
Você reutilizou bastante as mesmas cores em diversos locais, quando for fazer isso é sempre bom criar uma variavel dentro do arquivo css e apenas trazer ela, pois assim fica mais facil editar o arquivo futuramente